OpcUaProjectBuilder4 generates an application which is not compiled

**Describe the bug**

OpcUaProjectBuilder4 generates a application which is not compiled

**To Reproduce**
Steps to reproduce the behavior:
1. Generate a project with the command
`OpcUaProjectBuilder4 testApp testApp 4840`

2. Try to build with:
`sh build.sh -t local `

3. Get an error:

```
ects/ASNeG/testApp/src/testApp/Library/Library.cpp
In file included from /home/flipback/projects/ASNeG/testApp/src/testApp/Library/Library.cpp:20:
/home/flipback/projects/ASNeG/testApp/src/testApp/Library/Library.h:27:11: error: unknown class name
      'ApplicationIf'; did you mean 'OpcUaStackServer::ApplicationIf'?
        : public ApplicationIf
                 ^~~~~~~~~~~~~
                 OpcUaStackServer::ApplicationIf
/home/flipback/.ASNeG/usr/include/OpcUaStack4/OpcUaStackServer/Application/ApplicationIf.h:55:18: note: 
      'OpcUaStackServer::ApplicationIf' declared here
        class DLLEXPORT ApplicationIf
                        ^
/home/flipback/projects/ASNeG/testApp/src/testApp/Library/Library.cpp:41:7: error: use of undeclared
      identifier 'Debug'
                Log(Debug, "Library::startup");
                    ^
/home/flipback/projects/ASNeG/testApp/src/testApp/Library/Library.cpp:48:7: error: use of undeclared
      identifier 'Debug'
                Log(Debug, "Library::shutdown");
                    ^
/home/flipback/projects/ASNeG/testApp/src/testApp/Library/Library.cpp:63:33: error: unknown type name
      'ApplicationIf'
extern "C" DLLEXPORT void  init(ApplicationIf** applicationIf) {
```

**Expected behavior**

An empty project must compile.

**Environment (please complete the following information):**
 - OS: Ubuntu 19.10
 - Version: Release4

--- a/src/OpcUaProjectBuilder/ProjectTemplate/src/ProjectName/Library/Library.cpp
+++ b/src/OpcUaProjectBuilder/ProjectTemplate/src/ProjectName/Library/Library.cpp
@@ -23,6 +23,9 @@
 #include <iostream>
 #include "BuildConfig.h"
 
+using namespace OpcUaStackCore;
+using namespace OpcUaStackServer;
+
 namespace ProjectName
 {
 

--- a/src/OpcUaProjectBuilder/ProjectTemplate/src/ProjectName/Library/Library.h
+++ b/src/OpcUaProjectBuilder/ProjectTemplate/src/ProjectName/Library/Library.h
@@ -23,19 +23,19 @@
 namespace ProjectName
 {
 
-	class DLLEXPORT Library
-	: public ApplicationIf
+	class Library
+	: public OpcUaStackServer::ApplicationIf
 	{
 	  public:
 		Library(void);
 		virtual ~Library(void);
 
 		//- ApplicationIf -----------------------------------------------------
-		virtual bool startup(void);
-		virtual bool shutdown(void);
-		virtual std::string version(void);
-		virtual std::string gitCommit(void);
-		virtual std::string gitBranch(void);
+		bool startup(void) override;
+		bool shutdown(void) override;
+		std::string version(void) override;
+		std::string gitCommit(void) override;
+		std::string gitBranch(void) override;
 		//- ApplicationIf -----------------------------------------------------
 
 	  private:

